pyproject.toml

```diff
@@ -76,6 +76,11 @@ dev = [
 #Repository = "https://url.to/repo"
 #"Bug Tracker" = "https://url.to/bug-tracker"
 #Changelog = "https://url.to/changelog"
+logo = [
+    "svg-py>=1.5.0",
+    "numpy>=2.1.3",
+    "rdflib>=7.1.1",
+]
```

Comment:
As discussed in person, I have a personal preference for much laxer dependency specifications. It makes the package more difficult to install in the presence of other libraries that, for example, haven't been updated to support the last version of everything, for example of `numpy`. So my suggestion here is to specify the minimal required version, if that is known in practice, or – even better in my opinion – not to specify a version at all. Should a problem with older versions arise, it can be fixed by defining a minimum version.
In general, at least within the scientific Python community, the requirement is to be very conservative with requiring newer versions. See for example:
https://scientific-python.org/specs/spec-0000/

All of that said, I am not going to fight for it, so feel free to leave it as-is. ;-)